plugins/pipelines/README.md

```diff
@@ -106,10 +106,15 @@ Only PR events of specified PR number will be searched when `prNum` is set
 
 `GET /pipelines/{id}/events?page={pageNumber}&count={countNumber}&sort={sort}&prNum={prNumber}`
 
-#### Get all jobs (including pull requests jobs)
-`archived` is optional and has a default value of `false`, which makes the endpoint not return archived jobs (e.g. closed pull requests)
+#### Get all jobs
 
-`GET /pipelines/{id}/jobs?archived={boolean}`
+Arguments:
+
+* `archived` - Optional and has a default value of `false`, which makes the endpoint not return archived jobs (e.g. closed pull requests)
+* `type` - Optional and can be set to `pr` or `pipeline` to only return PR jobs or non-PR jobs
+* `jobName` - Optional and can be set to only return only a single job
```

Comment:
What's the use case for this param ? is this a single job ? or does it match multiple ? 

won't it be ideal for this to be a path param ?

Reply:
https://github.com/screwdriver-cd/screwdriver/issues/1522